ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,20 +32,20 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
-    public void EndVisitTest() throws Exception {
+    public void endVisitTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
         ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage = findPatientPage.clickOnFirstPatient();
         PatientVisitsDashboardPage patientVisitsDashboardPage = clinicianFacingPatientDashboardPage.goToRecentVisits();
-        patientVisitsDashboardPage = patientVisitsDashboardPage.endVisit();
```

Comment:
The reason of refactoring this method can be located [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/497/commits/f32ade42c62bf5f39ea2237660efff3164a4d2af)